docs/content/preview/tutorials/azure/azure-app-service.md

```diff
@@ -0,0 +1,140 @@
+
+As we're deploying our application to Azure from a local repository on our machine, we can forego all settings related to continuous deployment and GitHub Actions. However, for production applications, these integrations greatly improve the development process by deploying applications to Azure automatically when the repository is updated, without the need for manual intervention.
+
+Additionally, by selecting a free or basic tier of Azure App Service, we will deploy our application to a single zone by default. Premium pricing plans allow for multi-zone deployments for increased availability.
+
+After creating your application, visit the associated URL to confirm it's up and running!
+
+![Application running in Azure and waiting for content.](/images/tutorials/azure/azure-app-service/azure-app-running.png "Application running in Azure and waiting for content.")
+
+Now that we've set the foundation on Azure, let's deploy a database cluster for YB Shoes on YugabyteDB Managed.
+
+## Get Started on YugabyteDB Managed
+
+Visit our documentation site to create a YugabyteDB cluster on Azure.
+
+I've chosen to deploy a three-node cluster to Azure, in the WestUS3 region. This configuration provides fault tolerance across availability zones. However, you can start with an always-free single-node cluster.
+
+![Create free YugabyteDB single-region cluster.](/images/tutorials/azure/azure-app-service/yb-create-cluster.png "Create free YugabyteDB single-region cluster.")
+
+Add the [outbound addresses for your web app](https://learn.microsoft.com/en-us/azure/app-service/overview-inbound-outbound-ips), which can be found in the networking tab of the Azure portal. This will ensure that a connection can be made between Azure App Service and YugabyteDB.
+
+![Locate the outbound IP addresses for your web app in Azure.](/images/tutorials/azure/azure-app-service/azure-outbound-traffic.png "Locate the outbound IP addresses for your web app in Azure.")
+
+Now that we have a working cluster in YugabyteDB Managed, let's add some data.
+
+## Add data to YugabyteDB
+
+Now that our cluster is running in the cloud, we can seed it with data using the provided `schema.sql` and `data.sql` files.
+
+1. Use the [YugabyteDB Cloud Shell](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-connect/connect-cloud-shell/)to connect to your cluster.
+1. Execute the `schema.sql` script against your cluster.
+1. Execute the `data.sql` script against your cluster.
+
+Now that your cluster is seeded with data, it's time to deploy and run the application on Azure.
+
+## Deploy YB Shoes to Azure App Service
+
+Azure provides multiple deployment methods for your applications.
+
+Since we'll be deploying our application from a local git repository, let's use the Azure CLI. It's also worth noting that Azure has a powerful extension for Visual Studio Code, which can execute the same commands.
```

Comment:
```suggestion
Since we'll be deploying our application from a local git repository, let's use the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/). It's also worth noting that Azure has a powerful extension for Visual Studio Code, which can execute the same commands.
```